recipients_app/lib/view/pages/dashboard_page.dart

```diff
@@ -75,32 +82,31 @@ class _DashboardView extends StatelessWidget {
 
     return BlocBuilder<PaymentsCubit, PaymentsState>(
       builder: (context, state) {
-        return Padding(
-          padding: AppSpacings.h8,
-          child: Column(
+        return RefreshIndicator(
+          key: _refreshIndicatorKey,
+          onRefresh: () async {
+            context.read<PaymentsCubit>().loadPayments();
+            context.read<SurveyCubit>().getSurveys();
+          },
+          child: ListView(
```

Comment:
Also here we have nested listview which seems to throw some errors when there are some surveys on home page